semantic_conventions/trace/database.yaml

```diff
@@ -382,6 +382,47 @@ groups:
           The collection being accessed within the database stated in `db.name`.
         examples: [ 'customers', 'products' ]
 
+  - id: db.elasticsearch
+    prefix: db.elasticsearch
+    type: span
+    extends: db
+    brief: >
+      Call-level attributes for Elasticsearch
+    attributes:
+      - id: doc_id
+        type: string
+        requirement_level:
```

Comment:
Nit/Non-blocking: I always think it's nice for consuming the conventions when the `Required` attributes come first in the table. When they are "mixed" I find it hard to read/interpret things.

Reply:
Yeah, the rendered table. Weird I thought the order would follow the yaml. Never mind then :)

Reply:
(Possibly a bug in the tooling, see https://github.com/open-telemetry/build-tools/issues/133#issuecomment-1428836054)